vaadin-crud-flow-parent/vaadin-crud-flow-integration-tests/src/test/java/com/vaadin/flow/component/crud/test/BasicUseIT.java

```diff
@@ -27,6 +31,14 @@ public void crudIsPresent() {
         Assert.assertNotNull($(CrudElement.class).first().getGrid());
     }
 
+    @Test
+    public void crudReplacesGrid() {
+        CrudElement crud = $(CrudElement.class).waitForFirst();
```

Comment:
NIT: the `crud` variable is never used, so it doesn't need to be declared.

Also, there are some unused imports.